app/frontend/components/pages/login/hardwareAuth.tsx

```diff
@@ -46,11 +47,12 @@ const LoadByHardwareWalletSection = () => {
             dangerouslySetInnerHTML={{__html: '&nbsp;'}}
           />
           <button
-            disabled={!ADALITE_CONFIG.ADALITE_ENABLE_TREZOR}
+            disabled={!ADALITE_CONFIG.ADALITE_ENABLE_TREZOR || isMobileOnly}
             {...tooltip(
               'Support for Trezor is temporarily disabled',
               !ADALITE_CONFIG.ADALITE_ENABLE_TREZOR
             )}
+            {...tooltip('Not supported on mobile devices', isMobileOnly)}
```

Comment:
the tooltip seems to stay there "forever" until I switch to another screen, don't we have some convenient way to make it only temporarily visible? :thinking: